setup.py

```diff
@@ -58,7 +58,7 @@ def desc():
         "Flask-JWT-Extended>=4.0.0, <5.0.0",
         "jsonschema>=3, <5",
         "marshmallow>=3.18.0, <4",
-        "marshmallow-sqlalchemy>=0.22.0, <0.29.0",
+        "marshmallow-sqlalchemy>=0.22.0, <2.0.0",
```

Comment:
IMO we should remove the upper bound altogether

```suggestion
        "marshmallow-sqlalchemy>=0.22.0",
```

to avoid issues like the one that Airflow ran into. since FAB is a library, its shouldn't impose such strict upper constraints on its dependencies and allow users to pin their own dependencies